extensions/amp-lightbox/1.0/component.jss.js

```diff
@@ -46,31 +46,19 @@ const wrapper = {
   height: '100%',
   position: 'fixed',
   boxSizing: 'border-box',
-};
-
-// User overridable styles
-const defaultStyles = {
   zIndex: 1000,
   backgroundColor: 'rgba(0, 0, 0, 0.9)',
   color: '#fff',
 };
 
-/* 
-  overflow: hidden does not trigger overscrollBehavior, so 
-  overflow: auto is applied even though the lightbox should not scroll.
-  We rely instead on the overflow: hidden in ContainWrapper to prevent 
-  scrolling inside the lightbox.
-*/
-const containScroll = {
-  overflow: 'auto', // Prevent scrolling inside lightbox.
-  overscrollBehavior: 'none', // Prevent scrolling outside lightbox.
+const content = {
+  overflow: 'auto !important',
```

Comment:
FMU: why shouldn't we have `overscrollBehavior: contain/none` here?

Reply:
From PR deploy bot you can see there is no scrolling that leaks to the underlying document from lightbox that have overflow (See first ex. with button "Open scrollable lightbox"): https://storage.googleapis.com/amp-test-website-1/amp_nomodule_8f7a2cb3fbeb19145d096204d9a114102c462130/examples/amp-lightbox.amp.html

This PR does not prevent lightboxes without overflowing content from scrolling on the outer document, however. And adding `overscrollBehavior` will not help that case.

Reply:
Discussed offline and I couldn't reproduce it because the mouse I was using :) Added `overscrollBehavior: none` here. 

Will separately re-introduce backdrop in a follow up PR.